src/extension/features/toolkit-reports/pages/balance-over-time/BalanceOverTime.jsx

```diff
@@ -109,6 +160,16 @@ export const BalanceOverTimeComponent = ({ allReportableTransactions, filters })
             onChange={() => setShouldGroupAccounts(!shouldGroupAccounts)}
           />
         </div>
+        {shouldGroupAccounts && (
+          <div className="tk-pd-x-1">
+            <LabeledCheckbox
+              id="tk-balance-over-time-groupaccounts-option"
+              checked={shouldGroupAccountsByType}
+              label="by type"
+              onChange={() => setShouldGroupAccountsByType(!shouldGroupAccountsByType)}
+            />
+          </div>
+        )}
```

Comment:
The thing that's been holding me up on this is that I'm currently refactoring our account/category filter modal to actually look more like a dropdown and then I was going to have you put these checkboxes into that dropdown but that's taking me a lot longer with how busy i've been IRL.

Going to go forward with this for now but if we add another checkbox, we're definitely going to need to revisit this UX.